planetarypraxis/smartforests
# Contributor refactor

This PR closes #201.

It also refactors the `ContributorMixin` logic:

- Remove the `Person` model, so all editors are now `User`, which makes it easy to...
- Materialise `ContributorMixin.contributors` to the database so that `User.edited_content_pages` can run efficiently with a single query

## How Can It Be Tested?

- Add and remove contributors from the sidebar of a few pages
- Look at a contributor's page and you should see their contributions change appropriately
  - Note that contributions are on a per-locale basis, not synced between different locale versions of a page

## Types of changes


## Checklist:
